nextflow.config

```diff
@@ -280,15 +279,25 @@ profiles {
     test_hichip   { includeConfig 'conf/test_hichip.config'   }
 }
 
+// Set default registry for Apptainer, Docker, Podman and Singularity independent of -profile
+// Will not be used unless Apptainer / Docker / Podman / Singularity are enabled
+// Set to your registry if you have a mirror of containers
+apptainer.registry   = 'quay.io'
+docker.registry      = 'quay.io'
+podman.registry      = 'quay.io'
+singularity.registry = 'quay.io'
+
+// Nextflow plugins
+plugins {
+    id 'nf-validation@0.2.1' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

Comment:
As we are working on some fixes in the plugin after the pipelines start testing it and finding bugs, it's better to not pin any version
```
plugins {
    id 'nf-validation' // Validation of pipeline parameters and creation of an input channel from a sample sheet
}
```
If you prefer to define a version, the current one is 0.3.0 (will probably be 0.3.1 very soon)

Reply:
The version number was removed.